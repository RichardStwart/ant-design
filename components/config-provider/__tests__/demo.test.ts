import demoTest from '../../../tests/shared/demoTest';

demoTest('config-provider', { only: ['className.tsx'] });
