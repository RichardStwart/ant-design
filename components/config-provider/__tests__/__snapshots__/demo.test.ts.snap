// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/config-provider/demo/className.tsx correctly 1`] = `
<button
  class="ant-btn ant-btn-default custom-btn"
  type="button"
>
  <span>
    Button
  </span>
</button>
`;
